### DISCUSSION TITLE
VisualizationAPI is missing 'listbox'

### DISCUSSION THREAD

**Discussion by szymontodavizlib** [DefinitelyTyped/DefinitelyTyped#56770] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/aa977cdb8b5120ae9dbe128b61356335cb071645/types/qlik-visualizationextensions/index.d.ts#L2236

Why no `listbox`? https://help.qlik.com/en-US/qlikview/May2021/Subsystems/Client/Content/QV_QlikView/List_Box.htm

[a comment by an automated account is not shown]

**Comment by samarmohan** [comment]
See #56799, copy and paste it into your own project and see if it works.
